Moshe

MosheBrizel opened the pull request:
i add a option to borrow #25

MosheBrizel committed:
test

BenZionHasida committed:
test to learn git

BenZionHasida committed:
test to learn git

BenZionHasida committed:
test to learn git

BenZionHasida committed:
test to learn git

BenZionHasida committed:
fix some spelling mistakes

BenZionHasida committed:
add a comments

MosheBrizel committed:
change the database
1.remove the number of the books and add a column of accompanied.
2. add a please to chack the database.

MosheBrizel committed:
change the database
1.remove the number of the books and add a column of accompanied.
2. add a please to chack the database.

MosheBrizel committed:
add the option to take a book with search the book

NoamAnisfeld reviewed (changes requested):
Thanks!

Can you please explain what you did? There're many changes and I'm not able to follow it easily

NoamAnisfeld committed:
Revert "add a comments"

This reverts commit d5bd6c4047831b2bb14598328f06f2c00520c3ad.

NoamAnisfeld committed:
Revert "fix some spelling mistakes"

This reverts commit 684c2e84e5ecb371a3054fc151c3ca56338eefbb.

NoamAnisfeld committed:
Revert "test to learn git"

This reverts commit c4de94abe9883c4e29bbf375a88a2747b774aefb.

NoamAnisfeld committed:
Revert "test to learn git"

This reverts commit 54ee03b08782d51d52deec9e82de2dd148118b5d.

NoamAnisfeld committed:
Revert "test to learn git"

This reverts commit 3726f7b38bfdf4d5718bdb22adda659360ebc08b.

NoamAnisfeld committed:
Revert "test to learn git"

This reverts commit 6696f62bdd6a9f6330695ea7d74255dd0511b334.

NoamAnisfeld committed:
Revert "test"

This reverts commit 6b40c38100561f9f09f29f12c60b7a46122efa8c.

NoamAnisfeld committed:
Revert "test"

This reverts commit 6b40c38100561f9f09f29f12c60b7a46122efa8c.

NoamAnisfeld commented:
Reverted the test commits and Ben Tzion's commits that don't belong here. 👆

MosheBrizel committed:
changes
1. an option to delete the database and to create the database.
2. add an option to change a column in a row in the database.

MosheBrizel committed:
sample changes

MosheBrizel committed:
Merge remote-tracking branch 'origin/Moshe' into Moshe

MosheBrizel committed:
change to borrow a book,
1. add a html page.
2. add a path in the flask page.
3. add a js file to control the all books on the clint page, and to send a post request to the server.
